charts/kargo/values.yaml

```diff
@@ -123,6 +123,8 @@ api:
     type: ClusterIP
     ## @param api.service.nodePort [nullable] Host port the `Service` will be mapped to when `type` is either `NodePort` or `LoadBalancer`. If not specified, Kubernetes chooses.
     # nodePort:
+    ## @param api.service.annotations Annotations to add to the api service. 
```

Comment:
```suggestion
    ## @param api.service.annotations Annotations to add to the API server's service. 
```

Reply:
With the change above, this LGTM. You do need to [run codegen](https://kargo.akuity.io/contributor-guide/hacking-on-kargo#executing-code-generation) after any edits to `values.yaml`.

If you prefer, I can run it for you.

Thanks!

Reply:
Thanks Kent. If you wouldn't mind running it. Go isn't a strength for me and I'd probably not do it right.